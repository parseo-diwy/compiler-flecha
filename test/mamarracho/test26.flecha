---- Primitiva: ADD

def printInt x =
  case x
  | Int -> (unsafePrintInt x; unsafePrintChar '\n')

def main =
  printInt (1 + 1);
  printInt (1 + 2);
  printInt (1 + 3);
  printInt (10 + 1);
  printInt (10 + 2);
  printInt (10 + 3);
  printInt (10 + 10);
  printInt (10 + 20);
  printInt (10 + 30)


