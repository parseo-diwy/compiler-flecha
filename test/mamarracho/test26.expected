2
3
4
11
12
13
20
30
40
